app/frontend/src/components/UserSummary.tsx

```diff
@@ -42,31 +42,33 @@ interface UserSummaryProps {
 export default function UserSummary({ children, user }: UserSummaryProps) {
   const classes = useStyles();
   return (
-    <div className={classes.root}>
-      <ListItemAvatar>
-        {!user ? (
-          <Skeleton variant="circle" className={classes.avatar} />
-        ) : (
-          <Avatar user={user} className={classes.avatar} />
-        )}
-      </ListItemAvatar>
-      <ListItemText
-        className={classes.titleAndBarContainer}
-        disableTypography
-        primary={
-          <Typography variant="h2" className={classes.title}>
-            {!user ? <Skeleton /> : `${user.name}, ${user.age}, ${user.city}`}
-          </Typography>
-        }
-        secondary={
-          <>
-            <ScoreBar value={(user?.communityStanding || 0) * 100}>
-              {COMMUNITY_STANDING}
-            </ScoreBar>
-            {children}
-          </>
-        }
-      />
-    </div>
+    <>
+      <div className={classes.root}>
+        <ListItemAvatar>
+          {!user ? (
+            <Skeleton variant="circle" className={classes.avatar} />
+          ) : (
+            <Avatar user={user} className={classes.avatar} />
+          )}
+        </ListItemAvatar>
+        <ListItemText
+          className={classes.titleAndBarContainer}
+          disableTypography
+          primary={
+            <Typography variant="h2" className={classes.title}>
+              {!user ? <Skeleton /> : `${user.name}, ${user.age}, ${user.city}`}
+            </Typography>
+          }
+          secondary={
+            <>
+              <ScoreBar value={(user?.communityStanding || 0) * 100}>
+                {COMMUNITY_STANDING}
+              </ScoreBar>
+            </>
+          }
+        />
+      </div>
+      {children}
```

Comment:
> Also changed the position of children in UserSummary so the Avatar icon doesn't take up the whole vertical space (and therefore the children have the whole width available)

Hmm not sure about this change if we are expecting anything in `children` to be dynamic. Might be better to [top align the avatar like how it's shown here](https://www.figma.com/file/z1aQUklCQewLsEFUo2HsLz/Couchers-Platform-Design?node-id=5156%3A45)? Ignore the fact the "moderator ..." text is above the bar in this design, this is where our dev "creative license" comes in to override their decision as supporting that would introduce inconsistencies/maintenance nightmares...

Reply:
It's really because at the moment we have two components in our flexboxed .root class. 
So `ListItemAvatar` is taking up the whole left hand side, and if we keep `children` in `ListItemText` 's secondary component then it's restricted to the right hand side only. But I can move it back in no problemo if you think it's better?

![image](https://user-images.githubusercontent.com/60824744/111071433-15df0980-84ce-11eb-8bd5-2626c93c536e.png)

Reply:
Yeah I think it's fine for the `ListItemAvatar` to take up the left hand side space, as we probably want the supplementary text/components to be aligned with whatever that's on the right. If we want a component to take up the full width/go below the avatar, I'd probably put this as an adjacent sibling component to the `UserSummary` component instead.